### DISCUSSION TITLE
docker部署的seatunnel跟dolphinscheduler，执行提示没目录

### DISCUSSION THREAD

**Discussion by jianchengwang** [apache/dolphinscheduler#15161] [Q&A]
`	/tmp/dolphinscheduler/exec/process/default/11594819722464/11594878361824_1/3/3/3_3.sh: line 4: /bin/seatunnel.sh: No such file or directory`

使用自带的集成插件是不是对目录有啥限制要求，针对我这种情况有解决方案吗

**Comment by CuiNing6** [comment]
解决了吗，我也遇到这个问题了

**Reply by jianchengwang** [reply]
`docker cp apache-seatunnel-${version}-bin.tar.gz docker-compose-dolphinscheduler-dolphinscheduler-worker-1:/opt/soft/
docker exec -it docker-compose-dolphinscheduler-dolphinscheduler-worker-1 bash -c "mkdir -p /opt/soft/seatunnel && tar -zxvf /opt/soft/apache-seatunnel-${version}-bin.tar.gz --strip-components 1 -C /opt/soft/seatunnel"
`
我目前是直接把seatunnel安装包拷贝到work容器里

**Comment by naturn** [comment]
可以考虑重新build work master 镜像，如果你使用的docker compose 可以修改docker-compose.yml 中work/master相关章节的配置增加build配置指向一个Dockerfile
